Description: Use the packaged python-mock package
 Upstream is "vendorizing" mock. This sux...
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-02-08

--- python-eventlet-0.16.1.orig/tests/db_pool_test.py
+++ python-eventlet-0.16.1/tests/db_pool_test.py
@@ -7,7 +7,8 @@ import os
 import traceback
 from unittest import TestCase, main
 
-from tests import mock, skipped, skip_unless, skip_with_pyevent, get_database_auth
+import mock
+from tests import skipped, skip_unless, skip_with_pyevent, get_database_auth
 from eventlet import event
 from eventlet import db_pool
 from eventlet.support import six
--- python-eventlet-0.16.1.orig/tests/websocket_test.py
+++ python-eventlet-0.16.1/tests/websocket_test.py
@@ -8,7 +8,8 @@ from eventlet.green import httplib
 from eventlet.support import six
 from eventlet.websocket import WebSocket, WebSocketWSGI
 
-from tests import certificate_file, LimitedTestCase, mock, private_key_file
+import mock
+from tests import certificate_file, LimitedTestCase, private_key_file
 from tests import skip_if_no_ssl
 from tests.wsgi_test import _TestBase
 
